content/en/user-guide/web-application/teams/index.md

```diff
@@ -1,27 +0,0 @@
----
-title: "Teams"
-tags: ["LocalStack Pro"]
-weight: 2
-description: >
-  LocalStack Organizations/Teams offer a flexible way of managing access roles, permissions, and API keys for your team members.
----
-
```

Comment:
I would prefer to not delete this page for now, and rather put up a deprecation notice. WDYT?

Reply:
not sure. I think the docs should only provide 1 way of doing things. there is no need to keep the old info around when it comes to license management

Reply:
I understand the point, but this won't be beneficial for existing users who have not been notified about this and would be confused about this. We have traditionally kept deprecated docs around (Cockpit vs Desktop app as an example), and removed them once we are sure that everyone has been notified of the change

Reply:
this page was made obsolete by the 'users and licences 'page